gnosis-safe-deploy/scripts/approveTransaction.js

```diff
@@ -0,0 +1,62 @@
+const EthersAdapter = require('@gnosis.pm/safe-ethers-lib').default;
+const Safe = require('@gnosis.pm/safe-core-sdk').default;
+const SafeServiceClient = require('@gnosis.pm/safe-service-client').default;
+const fortaAgent = require('forta-agent');
+
+const { ethers } = fortaAgent;
```

Comment:
Is there a reason why we need `ethers` from the `forta-agent` package and not the `ethers` package?

Reply:
No, just reusing code from the propose transaction script, where we needed to bring in the `forta-agent` package so we used the `ethers` that it exports.

Fixed.